src/python/pyLCIO/__init__.py

```diff
@@ -1,4 +1,4 @@
-from base.SetupLcioDictionary import setupLcioDictionary
```

Comment:
Did you try removing this to see if the project structure remains? It will probably fix the need to have a . in front of base.

Reply:
I didn't touch anything like this manually. Everything seemed to work once I ran the conversion tool.

Reply:
I guess, I see the need of a . as pointing to a bug.  Not a big deal and doesn't need to be fixed now.